zeeguu/api/endpoints/feature_toggles.py

```diff
@@ -51,6 +52,13 @@ def _tiago_exercises(user):
     return right_user and right_language
 
 
+def _merle_exercises(user):
+    # Change this to Match Merle's Exercises
+    # 534: Mircea, 4089: Tiago Test User, 4192: Merle's User
```

Comment:
I guess if we don't put this comment on GH, then there is a lower chance that somebody can identify our data in the anonymized DB :)